cypress.config.js

```diff
@@ -0,0 +1,58 @@
+const yaml = require('js-yaml');
```

Comment:
Won't take credit for this as most of this code was generated by @davidding, thanks for being awesome 👍🏻